Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/ImageMtl.mm

```diff
@@ -20,6 +20,30 @@
 namespace rx
 {
 
+namespace
+{
+angle::FormatID intendedFormatForMTLTexture(id<MTLTexture> texture,
```

Comment:
@djg do you know if this was a change you added or a bad merge on my part?

Reply:
did you sort this out?

Reply:
I did :) it is needed, so correct to keep the change